km_api/know_me/migrations/0020_kmuseraccessormodel.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.3 on 2017-08-08 17:56
+from __future__ import unicode_literals
+
+from django.conf import settings
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import permission_utils.model_mixins
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('know_me', '0019_emergencycontact'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='KMUserAccessor',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('accepted', models.BooleanField(default=False, help_text='The KMUser has accepted the access.', verbose_name='is accepted')),
+                ('can_read_everywhere', models.BooleanField(default=False, help_text='The user has can read everywhere access.', verbose_name='read everywhere')),
+                ('can_write_everywhere', models.BooleanField(default=False, help_text='The user has write everywhere access.', verbose_name='write everywhere')),
+                ('km_user', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='km_user_accessors', related_query_name='km_user_accessor', to='know_me.KMUser', verbose_name='Know Me user')),
+                ('user_with_access', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='km_user_accessors', related_query_name='km_user_accessor', to=settings.AUTH_USER_MODEL, verbose_name='user')),
+            ],
+            options={
+                'verbose_name_plural': 'know me user accessors',
+                'verbose_name': 'kmow me user accessor',
+            },
+            bases=(permission_utils.model_mixins.IsAuthenticatedMixin, models.Model),
+        ),
+        migrations.AlterField(
+            model_name='emergencycontact',
+            name='alt_phone_number',
+            field=models.CharField(blank=True, max_length=15, validators=[django.core.validators.RegexValidator(code='invalid_phone_number', message='Enter valid phone number', regex='^1?\\d{9,15}$')], verbose_name='alternate phone number'),
+        ),
+        migrations.AlterField(
+            model_name='emergencycontact',
+            name='phone_number',
+            field=models.CharField(max_length=15, validators=[django.core.validators.RegexValidator(code='invalid_phone_number', message='Enter valid phone number', regex='^1?\\d{9,15}$')], verbose_name='phone number'),
+        ),
```

Comment:
These `AlterField` migrations shouldn't be a part of this pull request. This is probably caused by #142.